pkg/controller/clusterpool/clusterpool_controller.go

```diff
@@ -78,6 +79,16 @@ func NewReconciler(mgr manager.Manager, rateLimiter flowcontrol.RateLimiter) *Re
 	}
 }
 
+func indexCDByClusterPool(o client.Object) []string {
```

Comment:
we don't seem to be testing/reusing this func.. so my personal preference would be to keep this inline.

Reply:
Done.

Though the CD controller's [indexClusterInstall](https://github.com/openshift/hive/blob/33be31582aa140ce772333d0437eb77cd723cee2/pkg/controller/clusterdeployment/clusterinstalls.go#L225) does the same.